stories/00-Foundations/font-styles.js

```diff
@@ -3,159 +3,159 @@ export default [
     text: "Header 10",
     styles: {
       fontFamily: 'var(--font-family-header)',
-      fontSize: 'var(--font-size-header-10)',
+      fontSize: 'var(--font-size-16)',
       fontWeight: 'var(--font-weight-header-10)',
-      lineHeight: 'var(--line-height-header-10)',
```

Comment:
I love how using the tokens directly in the docs means we catch this stuff when we refactor

Reply:
👍